apps/tutorial-servo-can-control/src/TutorialServo.cpp

```diff
@@ -0,0 +1,53 @@
+//
+//  TutorialServo.cpp
+//  
+//
+//  Created by Ari Wasch on 9/18/20.
+//
+
+#include "TutorialServo.h"
+#include "mbed.h"
+
+TutorialServo::TutorialServo(PinName servoPin, float servoRangeInDegrees = 180.0, float minPulsewidthInMs = 1, float maxPulsewidthInMs = 2) : _pin(pin) {
```

Comment:
Remove _pin, it's not a member variable in this class. Also remove the default values from the function definition, they should only be given in the function declaration (your header file)